synapse/swarm/runtime.py

```diff
@@ -144,10 +144,17 @@ def getTufosByPropFrom(self, prop, valu=None, limit=None, mintime=None, maxtime=
 
         dyntask = gentask('getTufosByProp',prop,valu=valu,limit=limit,mintime=mintime,maxtime=maxtime)
 
-        for svcfo,retval in self.callByTag(fromtag,dyntask):
-            for tufo in retval:
-                tufo[1]['.from'] = svcfo[0]
-                yield tufo
+        timeout = None
```

Comment:
So, i totally understand the idea here, *but*...  we may want to consider making the use of callByTag more "resiliant" to partial failure cases by documenting the timeout in the results but still returning whatever we managed to get ( takes us from "one box being borked takes out the whole thing" to "one box being dorked makes just that data unavailable" )

thoughts?